pkg/webhook/webhook.go

```diff
@@ -531,43 +583,56 @@ func MutateHandler(w http.ResponseWriter, req *http.Request) {
 			var hugepageResourceList []hugepageResourceData
 			glog.Infof("injectHugepageDownApi=%v", injectHugepageDownApi)
 			if injectHugepageDownApi {
-				for _, container := range pod.Spec.Containers {
+				for containerIndex, container := range pod.Spec.Containers {
```

Comment:
Look the resource patch creation logic still considers pod with single container. shouldn't we fix that too for multiple containers ?

Reply:
I created Issue #44 to track it. Looks like the problem has been in  the code for a while. Not sure I have time at the moment to fix, but it definitely needs to be fixed. I created a podspec manually to make sure the hugepages worked in the multi-container environment.